com.unity.ml-agents/Runtime/Inference/BarracudaModelParamLoader.cs

```diff
@@ -149,6 +149,16 @@ public static string[] GetOutputNames(Model model)
                 return failedModelChecks;
             }
 
+            foreach (var constantName in TensorNames.RequiredConstants)
```

Comment:
This prevents null-reference exceptions when trying to load models without the expected constants in them (which was happening for onnx files before a fix on master). This can be moved to another PR, but it's a nice to have.